### PR TITLE
Separates Rock The Vote Import into two jobs to enable larger file imports

### PR DESCRIPTION
**Pull request by chloealee**
#### What's this PR do?
Separates Rock The Vote Import into two jobs to enable larger file imports.
 - One job to process CSV.
 - One job to create post in Rogue.

#### How should this be reviewed?
- Does all the logic seem ok? 

#### Any background context you want to provide?
- Unfortunately, we will have to get rid of the stats when we separate these jobs. Are we ok with this for now? We should still have all the info. in the logs though! 

#### Relevant tickets

Fixes [this](https://www.pivotaltracker.com/n/projects/2019429/stories/159660724) Pivotal Card.

### CODE REVIEW

File: app/Jobs/CreateTurboVotePostInRogue.php [hunk 1]
```diff
@@ -55,7 +54,7 @@ public function handle(Rogue $rogue)
                 ]);
             }
 
-            if ($user) {
+            if (isset($user)) {
```
[the remark this answers is not in the document]
Comment (katiecrane) [reply app/Jobs/CreateTurboVotePostInRogue.php]: good catch! thanks for explaining :)

File: app/Jobs/ImportRockTheVotePosts.php [hunk 2]
```diff
@@ -26,12 +26,10 @@ class ImportRockTheVotePosts implements ShouldQueue
     protected $filepath;
 
     /**
-     * The path to the stored csv.
+     * The total records in the stored csv.
```
Comment (katiecrane) [inline app/Jobs/ImportRockTheVotePosts.php]: is this a count of the total records or is it the records themselves?

File: app/Jobs/CreateRockTheVotePostInRogue.php [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+                        'source' => 'rock-the-vote',
+                        'source_details' => $sourceDetails,
+                        'details' => $postDetails,
+                    ];
+
+                    try {
+                        $post = $rogue->createPost($postData);
+
+                        if ($post['data']) {
+                            info('post created in rogue for ' . $this->record['id']);
```
Comment (katiecrane) [inline app/Jobs/CreateRockTheVotePostInRogue.php]: there's no "id" column in RTV csvs, I believe previously we were using email address?

**Comment by katiecrane** [comment]
@chloealee it's really hard to tell if any of the logic changed in the fresh RTV file, I'm trying to figure out a good way to compare it to the old logic - do you know if you changed anything after you copied and pasted?

File: app/Jobs/ImportRockTheVotePosts.php (hunk 2 above)
Comment (chloealee) [reply app/Jobs/ImportRockTheVotePosts.php]: @katiecrane great q! This is [the count](https://github.com/DoSomething/chompy/blob/7b32036b779ce702f50076bc1e60c8c87c75554f/app/Traits/ImportToRogue.php#L35) of the total records. Updated this comment to make it more clear!

File: app/Jobs/CreateRockTheVotePostInRogue.php (hunk 3 above)
Comment (chloealee) [reply app/Jobs/CreateRockTheVotePostInRogue.php]: ah great catch! Updated :)

**Comment by chloealee** [comment]
@katiecrane ah yes it is hard to tell ! You could maybe compare to [this file](https://github.com/DoSomething/chompy/blob/master/app/Jobs/ImportRockTheVotePosts.php)?

**Comment by chloealee** [comment]
@katiecrane the only thing that I added was that log line that you caught before that had `$record['id']` instead of `$record['Email address']` and deleted the stats stuff since it won't work to have in separate two jobs!

**Comment by katiecrane** [comment]
@chloealee okay I think we're good then! Let's just make sure to do some spot checking on QA just in case! great work!

**Comment by chloealee** [comment]
yes definitely @katiecrane ! :)

**Pull request merged**